gulpfile.js

```diff
@@ -465,7 +468,21 @@ function deployCesium(bucketName, uploadDirectory, cacheControl) {
         })
         .then(function() {
             console.log('Cleaned ' + existingBlobs.length + ' files.');
+        })
+        .catch(function(error) {
+            errors.push(error);
         });
+    }).then(function() {
+        if (errors.length === 0) {
+            done();
+            return;
+        }
+
+        console.log('Errors: ');
+        errors.map(function(e) {
+            console.log(e);
+        });
+        done(1);
```

Comment:
Would that work to fail the task since I'm not actually returning the promise at the task level?